Disable v8 stack traces to restore signal handling

konradreiche opened the pull request:
This fixes https://github.com/augustoroman/v8/issues/38 which is more severe that it looks like in the issue. With v8's debug stack trace dumping turned on, the Go signal handling stops working. Go's panic mechanism relies on the `SIGSEGV` process signal. With v8's stack trace dumping the signal handler of the Go runtime is completely skipped.

This means that dereferencing any nil pointer in Go will result in a C stack trace with no information about the error happening in the Go code whatsoever.

What is worse, **Go's formatter intentionally panics when printing nil pointers** in order to recover from the panic and print it as `<nil>` eventually.

Without this change, once v8 is initialized, printing nil pointer will result in a crash. Running this v8 wrapper without this change in a product environment is highly risky.

You can read in more detail about this here http://konradreiche.com/posts/a-cautionary-tale-of-cgo-embedding.html

augustoroman commented on v8_c_bridge.cc:
```diff
@@ -168,7 +168,7 @@ extern "C" {
 
 void v8_init() {
-  v8::Platform *platform = v8::platform::CreateDefaultPlatform();
+  v8::Platform *platform = v8::platform::CreateDefaultPlatform(0, v8::platform::IdleTaskSupport::kDisabled, v8::platform::InProcessStackDumping::kDisabled);
```
```suggestion
  v8::Platform *platform = v8::platform::CreateDefaultPlatform(
      0, // thread_pool_size
      v8::platform::IdleTaskSupport::kDisabled,
      v8::platform::InProcessStackDumping::kDisabled);
```

konradreiche commented:
Thanks! Just added the regression test. I hope comments are in line with your style. I am only checking that it should throw a runtime error, not the exact string. Also added the style changes for the invocation to the v8 constructor.
